Changelog.md

```diff
@@ -1,5 +1,10 @@
 ## Ignition Physics 1.x
 
+### Ignition Physics 1.10.0 (2020-11-04)
+
+1. Ignore invalid joint commands in the dartsim-plugin
+    * [Pull request 137](https://github.com/ignitionrobotics/ign-physics/pull/137)
```

Comment:
Missing a couple of PRs:

https://github.com/ignitionrobotics/ign-physics/compare/ignition-physics_1.9.0...ign-physics1